src/basics.md

```diff
@@ -142,10 +163,35 @@ use rand::Rng;
 
 fn main() {
     let mut rng = rand::thread_rng();
-    println!("{}", rng.gen_range(0, 10));
+    println!("Integer: {}", rng.gen_range(0, 10));
+    println!("Float: {}", rng.gen_range(0.0, 10.0));
 }
 ```
 
+Alternatively, one can use the [`Range`](https://doc.rust-lang.org/rand/rand/distributions/range/struct.Range.html) distribution.
```

Comment:
Disagree. If someone wants to generate numbers within a range, they'll follow that link, and it's fair to show them the two ways of doing it there. If they don't understand the second example, no real problem since they already have an alternative.

On the other hand, understanding how to do that given just the normal distribution example may not be so obvious.